rust/connlib/connection/src/pool.rs

```diff
@@ -30,6 +31,9 @@ use stun_codec::rfc5389::attributes::{Realm, Username};
 // Note: Taken from boringtun
 const HANDSHAKE_RATE_LIMIT: u64 = 100;
 
+/// How often wireguard will send a keep-alive packet.
+pub(crate) const WIREGUARD_KEEP_ALIVE: u16 = 5;
```

Comment:
would it make sense to expose this to upper layers? That way in the future we can configure it differently to conserve batter

Reply:
Well, it is a bit tricky! Our detection of missed keep-alives only works if we know what the _other_ peer's keep-alive is set to. Hence, I didn't want to make this a config option because it will break too easily if we change this across versions.

I can add a comment about this to the constant if you want. If we want this to be configurable, then we should probably sync it via the portal so we know what keep-alive the other peer is going to use.

I am all for conserving battery but I think we should measure that first so we can make targeted optimizations.